sdk/eventhub/azure-eventhub/TROUBLESHOOTING.md

```diff
@@ -9,20 +9,27 @@ This troubleshooting guide covers failure investigation techniques, common error
 - [Connectivity issues](#connectivity-issues)
   - [Timeout when connecting to service](#timeout-when-connecting-to-service)
   - [SSL handshake failures](#ssl-handshake-failures)
-  - [Socket exhaustion errors](#socket-exhaustion-errors)
+  - [Managing clients](#managing-clients)
+  - [Adding components to the connection string does not work](#adding-components-to-the-connection-string-does-not-work)
+    - ["TransportType=AmqpWebSockets" Alternative](#transporttypeamqpwebsockets-alternative)
```

Comment:
I think we should probably change ours to AmqpOverWebsocket, as per our naming

Reply:
Both this line and the one below should be changed to match the section title so the link is clickable.

Reply:
Same thing for a few other section titles